### DISCUSSION TITLE
Error when serveing and building site. "YAML Exception reading C:/Users/user/my-site/_pages/about.md: (<unknown>): did not find expected key while parsing a block mapping at line 2 column 1"

### DISCUSSION THREAD

**Discussion by Hankenoj** [alshedivat/al-folio#1873] [Q&A] [answered]
### Discussed in https://github.com/alshedivat/al-folio/discussions/1857

<div type='discussions-op-text'>

<sup>Originally posted by **Hankenoj** October 20, 2023</sup>
Hello and thank you in advance. I get this error when I serve mysite and build

YAML Exception reading C:/Users/user/my-site/_pages/about.md: (<unknown>): did not find expected key while parsing a block mapping at line 2 column 1

I have my Confing yaml and about.md attached I couldnt find any issues myself with them. This seems to be messing with my about page cause when I serve my site and check it on local host I get this. Any way to solve this issue and get my about page working again? Let me know if I need to give more information I am new to makeing sites and jekyll.

![badindexscreenshot](https://github.com/alshedivat/al-folio/assets/142030543/77780816-f6b7-419e-9485-12c0ce7f2828)

about.md file 

`---
layout: about
title: About
permalink: /
subtitle: "Ye have not because ye ask not" James 4:2 (KJV)

profile:
  align: right
  image: Profileimg.jpg
  image_circular: true # crops the image to make it circular
  address: ""

news: false # includes a list of news items
latest_Projects: true  # includes a list of the newest posts
selected_papers: false # includes a list of papers marked as "selected={true}"
social: false # includes social icons at the bottom of the page
---`

Yaml file as a txt cause gitup dont support yaml's

[_config.txt](https://github.com/alshedivat/al-folio/files/13059854/_config.txt)

</div>

**Comment by george-gca** [comment, marked as answer]
Check the tabulation and also the case of the words in your file. For example, it should be:

```yaml
profile:
  align: right
  image: Profileimg.jpg
  image_circular: true # crops the image to make it circular
  address: ""
```
since all these informations are for the `profile`.

Also, yours is `latest_Projects` when it should be `latest_projects`. Actually I don't believe this field `latest_projects` is supported.

**Comment by Hankenoj** [comment]
That fixed it thank you.
